src/frontend/posts_page/components/TextEditorSetting.tsx

```diff
@@ -1,15 +1,12 @@
 import React from 'react';
 
-interface TextEditorProps {
-}
-
-interface TextEditorState {
+interface ITextEditorState {
   rich: boolean,
   plain: boolean,
 }
 
-class SetTextEditor extends React.Component<TextEditorProps, TextEditorState> {
-  constructor(props: TextEditorProps) {
+class TextEditorSetting extends React.Component<{}, ITextEditorState> {
```

Comment:
Reason for renaming: Components shouldn't have verb as a class name. `SetTextEditor` is more of a action name.